### PR TITLE
feature/update_df

### PR DESCRIPTION
**Pull request by Ksteeds**
Able to handle DataFrames with all possible additional metrics: ['grading', 'eop_lag', 'mop_lag'].

### CODE REVIEW

**Comment by Ksteeds** [comment]
Redesign required to handle updating the original DataFrame on a ticker level.

**Comment by Ksteeds** [comment]
Unit Test still required.

File: macrosynergy/management/update_df.py [hunk 1]
```diff
@@ -13,43 +14,94 @@ def update_df(df: pd.DataFrame, df_add):
     :param <pd.DataFrame> df_add: dataframe with the latest values. The categories will
         either be appended to the aggregate dataframe, or replace the previously defined
```
Comment (rsueppel) [inline macrosynergy/management/update_df.py:15]: :param <pd.DataFrame> df_add: dataframe that will be appended to existing standardized dataframe and replace any existing tickers, combinations of cross sections and categories with the same names.

File: macrosynergy/management/update_df.py [hunk 2]
```diff
@@ -13,43 +14,94 @@ def update_df(df: pd.DataFrame, df_add):
...
         category.
+    :param <bool> tickers: if the original dataframe should be updated on a individual
+        ticker level, as opposed to an entire panel being changed, set the parameter to
+        True. Default is False and the complete category will be updated.
```
[the remark this answers is not in the document]
Comment (Ksteeds) [reply macrosynergy/management/update_df.py]: In principle it has been resolved and DocStrings updated to reflect the change.

**Comment by Ksteeds** [comment]
Deleted branch.

**Pull request closed without merging**